job-simulation.rb

```diff
@@ -1,4 +1,57 @@
 require './Stack.rb'
 require './Queue.rb'
 
+  @waiting = Queue.new
+  @workers = Stack.new
 
+  @workers.push("one")#co. starts with six hires for 3 months
+  @workers.push("two")
+  @workers.push("three")
+  @workers.push("four")
+  @workers.push("five")
+  @workers.push("six")
```

Comment:
Pushing these as words is definitely harder. Instead, you could have said:

```
6.times do |i|
  @workers.push(i)
end
```

or perhaps 

```
(1..6).each do
```

^^ I think that one works? Maybe?